source/ouibounce.js

```diff
@@ -123,11 +123,18 @@ function ouibounce(el, custom_config) {
     _html.removeEventListener('keydown', handleKeydown);
   }
 
+  var supressed;
```

Comment:
This should probably be up with the other variables near the top and initialized to false.